modules/serverless-workflow-basic.adoc

```diff
@@ -55,40 +63,33 @@ The service's environment variable `TARGET` is now set to `Kn`.
 +
 ----
 $ kn service describe hello
+Name:       hello
+Namespace:  default
+Age:        13m
+URL:        http://hello.default.apps-crc.testing
+Address:    http://hello.default.svc.cluster.local
 
-Name:         hello
-Namespace:    default
-URL:          http://hello.default.apps-crc.testing
-Address:      http://hello.default.svc.cluster.local
-Annotations:  serving.knative.dev/creator=kube:admin, serving.knative.dev/lastModifier=kube:admin
-Age:          34m
-
-Revisions:
-  100%  Name:    hello-fszsh-2 [2] (35s)
-        Image:   gcr.io/knative-samples/helloworld-go (pinned to 5ea96ba4b872)
-        Env:     TARGET=Kn
-        Memory:  100M ... 200M
-        CPU:     400m ... 1
-
-Conditions:
+Revisions:  
+  100%  @latest (hello-dghll-2) [2] (1m)
+        Image:  gcr.io/knative-samples/helloworld-go (pinned to 5ea96b)
+
+Conditions:  
   OK TYPE                   AGE REASON
-  ++ ConfigurationsReady    22s 
-  ++ Ready                  22s 
-  ++ RoutesReady            22s ribe hello
+  ++ Ready                   1m 
+  ++ ConfigurationsReady     1m 
+  ++ RoutesReady             1m 
 ----
 
 . Delete the service.
 +
 ----
 $ kn service delete hello
-
 Service 'hello' successfully deleted in namespace 'default'.
 ----
 +
 You can then verify that the 'hello' service is deleted by attempting to `list` it.
```

Comment:
Consider marking up `hello`?

Reply:
Thanks for the review @adellape! updated the PR.